lib/lax.js

```diff
@@ -30,6 +30,65 @@ function _arrayWithHoles(arr) { if (Array.isArray(arr)) return arr; }
     var lastY = 0;
     var currentBreakpoint = 'default';
     var breakpointsSeparator = "_";
+    /*
+     * Easing Functions - inspired from http://gizma.com/easing/
+     * only considering the t value for the range [0, 1] => [0, 1]
+     */
+
+    var easings = {
```

Comment:
Why you not use a ready-made library?
e.g. https://github.com/mattdesl/eases

Reply:
At this moment library without any dependencies, dont want to add some of them